src/protocols/rdp/rdp_settings.h

```diff
@@ -359,6 +359,14 @@ typedef struct guac_rdp_settings {
      * the destination directory is otherwise ambiguous).
      */
     char* sftp_directory;
+
+    /**
+     * The interval at which SSH keepalive messages are sent to the server for
+     * SFTP connections.  The default is 0 (disabling keepalives), and a value
+     * of 1 is automatically increased to 2 by libssh2 to avoid busy loop corner
+     * cases.
+     */
+    unsigned sftp_server_alive_interval;
```

Comment:
How does it look, now?  It isn't the cleanest, because 0 is actually valid (to disable keepalive).  So, I'm setting negative values to 0 (since negative values don't make sense), and then just warning that 1 gets rounded up to 2 and allowing libssh2 to do that internally.